### ISSUE TITLE
Re-enable ESLint Tests

### ISSUE THREAD

**Issue by kappanjoe** [OriOri-CCP7/oriori#41]
- [x] Remove the following rule overrides from `client/package.json`
  - [x] "no-unused-vars"
  - [x] "exhaustive-deps"
  - [x] "no-empty-patterns"
- [x] Do not merge into production until after June 7th

**Comment by kappanjoe** [comment]
Resolved.
